ha/core/event_manager/event_manager.py

```diff
@@ -318,6 +318,19 @@ def get_events(self, component : str) -> list:
             list: List of events.
         """
 
+        key = f'{const.COMPONENT_KEY}/{component}'
```

Comment:
key = f'{const.COMPONENT_KEY}/{component}'

for all key use _prepare_key()
all keys should be in one place to make it easy to maintain

Reply:
key creation here is based on the # defines done in earlier PRs.
We can have a generic key creation for all functions , not just get_events and message_type

That can be done in this PR or separately @ajay-paratmandali @indrajitzagade any suggestions  ?

Reply:
Yes, I agree. we need generic key creation.